File: reverse_strings/reverse_strings.py
def reverse_strings(my_phrase):
    my_phrase_as_array = list(my_phrase)
    new_phrase = list()
    
    for index in reversed(my_phrase_as_array):
        new_phrase.append(index)

    return new_phrase

print(reverse_strings("Hola mundo"))

File: reverse_strings/reverse_strings.php
<?php
    function reverseString($phrase) {
        $phraseAsArray = str_split($phrase);
        $reversePhrase = [];
        
        for ($index = count($phraseAsArray) -1; $index >= 0; $index--) {
            $reversePhrase[] = $phraseAsArray[$index];
        }
        
        return implode("", $reversePhrase);
    }
    
    echo reverseString("Hola mundo"); //odnum aloH
?>
